backend/routes/prisonerProfileRouter.js

```diff
@@ -18,7 +18,8 @@ const prisonerIncentiveLevelDetails = require('../controllers/prisonerProfile/pr
 
 const prisonerProfileServiceFactory = require('../services/prisonerProfileService')
 const personServiceFactory = require('../services/personService')
-const paginationService = require('../services/paginationService')
+const paginationService = require('../services/pagination2Service')
```

Comment:
pagination2Service is a temp name at the moment as we're not sure if we want to make this the default pagination service.